Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -151,10 +157,24 @@ open class MessageContentCell: MessageCollectionViewCell {
         let topMessageLabelText = dataSource.messageTopLabelAttributedText(for: message, at: indexPath)
         let bottomMessageLabelText = dataSource.messageBottomLabelAttributedText(for: message, at: indexPath)
 
+        let sentDate = message.sentDate
+        let sentDateString = MessageKitDateFormatter.shared.string(from: sentDate)
+        let timeLabelFont : UIFont = .boldSystemFont(ofSize: 10)
+        var timeLabelColor: UIColor = .darkGray
+        if #available(iOS 12.0, *), traitCollection.userInterfaceStyle == .dark {
+            timeLabelColor = .white
```

Comment:
Still not sure I like this... 😄 Could you maybe please try to play with `systemGray` or any similar color that adopts automatically?

https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/